Config/DefaultGame.ini

```diff
@@ -4,10 +4,10 @@
 ProjectID=0DA629C640F9BB551736EC95C1359C90
 Description=Memory game as mod.io demo
 CopyrightNotice=Copyright (c) 2023 mod.io
-CompanyName=mod.io
+CompanyName=

```

Comment:
Out of curiosity why did you remove the company name?

Reply:
Microsoft platforms (GDK, XB1, XSX) were failing to compile due to that name not meeting format requirements (tried `modio` too), I suspect it might be to do with developer ID or something, judging by the error in question.

No real reason we can't re-add it though, as I realise this doesn't *need* to compile for consoles.

Reply:
Ah ok, all good was just curious. LGTM